packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -7,19 +7,33 @@
  * More information about configuration can be found at:
  *
  * https://trufflesuite.com/docs/truffle/reference/configuration
+ * 
+ * Hands-off deployment with Infura
+ * --------------------------------
+ *
+ * Is your application complex and requires lots of transactions to deploy?
+ * Use this appproach to make deployment a breeze 🏖️:
  *
  * To deploy via Infura you'll need a wallet provider (like @truffle/hdwallet-provider)
  * to sign your transactions before they're sent to a remote public node. Infura accounts
- * are available for free at: infura.io/register.
+ * are available for free at 🔍: https://infura.io/register
  *
  * You'll also need a mnemonic - the twelve word phrase the wallet uses to generate
- * public/private key pairs. If you're publishing your code to GitHub make sure you load this
- * phrase from a file you've .gitignored so it doesn't accidentally become public.
- *
- * It is recommended to store your secret variables in a .env file (which you should include in .gitignore).
- * In your project root, run `$ npm install dotenv`.
- * Create .env and declare your MNEMONIC and infura PROJECT_ID variables inside.
- * .gitignore is already added with this project with .env in it.
+ * public/private key pairs. You can store your secret 🤐 variables in a .env file. In your 
+ * project root, run `$ npm install dotenv`. Create .env (which should be .gitignored) 
+ * and declare your MNEMONIC and infura PROJECT_ID variables inside.
+ * 
+ * PLEASE NOTE 🗒️: This is an insecure workflow! It is possible through human error to leak 
+ * your mnemonic on Github. 
+ * 
+ * Deployment with Truffle Dashboard
+ * ---------------------------------
+ * 
+ * Are you concerned about the security 🤔? Use this approach to get rid of this hassle 🤦‍♀️:
+ * 
+ * A more secure 🔒 workflow is to use truffle dashboard which leverages
+ * Metamask for signing your transactions and does not require you to reveal your mnemonic to Truffle.
```

Comment:
The first sentence isn't so clear about what _the_ security and _this_ hassle are. Perhaps this section can be more succinct? Something like:

Are you concerned about security? Truffle dashboard lets you review transactions in detail, and leverages MetaMask for signing so there's no need to copy-paste your mnemonic.

Of course you can insert emojis to your heart's content 🦄